.github/CODEOWNERS

```diff
@@ -100,6 +100,7 @@
 /neoload/                                @Neotys-Labs/r-d @DataDog/ecosystems-review
 /neutrona/                               @DavidFlamini david@neutrona.com
 /nextcloud/                              @eplanet
+/ngrok/                                  @mkarnowski @DataDog/ecosystems-review
```

Comment:
Could you also please add entries under `# Community Partners + Documentation` (see `neoload` as an example)?

Reply:
CODEOWNERS updated